### PR TITLE
Feature/fixes dx29 dx30 dx33 dx34

### PR DESCRIPTION
**Pull request by W3stside**
## JIRA Ticket fixes

1. DX-29: _MenuWallet needs to accurately reflect ether price_ 
 --> in progress, dealing w/contract issues
2. DX-30: _change wording "available balance" to your current balance in the receiving tokens_
 --> done
3. DX-33: _consider submission button changing to another wording_
 --> done (up to changes though)
4. DX-34: _Disallow getting to Submit Sell Order (last screen) if no sell amount entered_
 --> done (may need more functionality such as letting user know why they cannot move forward)

### CODE REVIEW

File: src/components/OrderPanel/index.tsx [hunk 1]
```diff
@@ -25,7 +26,7 @@ const OrderPanel: React.SFC<OrderPanelProps> = ({ sellToken, buyToken }) => (
     <AuctionPriceBar header="Closing Price" />
     <AuctionSellingGetting />
     {/* TODO: replace onclick with some logic (maybe: "to" prop) */}
-    <ButtonCTA onClick={() => console.log('Continuing to wallet')} to="/wallet">
+    <ButtonCTA onClick={() => console.log('Continuing to wallet')} to={+sellAmount > 0 && '/wallet'}>
```
Comment (Velenir) [inline src/components/OrderPanel/index.tsx]: You know what would be nice for indication that we can't go further, `disabled={sellAmount>0}`. Then pass this disabled all the way through to `<a disabled>` and change its style with `a.class[disabled]`.

File: src/components/TokenItem/index.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const TokenItem: React.SFC<TokenItemProps> = ({ onClick, ...rest }) => {
       {mod && <strong>{mod2Title[mod] || mod}</strong>}
       <i data-coin={code}></i>
       <big>{name}</big><code>{code}</code>
-      <small>{mod && 'AVAILABLE'} BALANCE:</small>
+      <small>{mod && (mod === 'sell' && 'AVAILABLE' || 'CURRENT')} BALANCE:</small>
```
Comment (Velenir) [inline src/components/TokenItem/index.tsx]: Why not use a ternary?

File: src/components/OrderPanel/index.tsx (hunk 1 above)
Comment (Velenir) [reply src/components/OrderPanel/index.tsx]: Also I'm not sure but when `<Link to={not location object or string}/>` there may be an error when you click it.

Comment (W3stside) [reply src/components/OrderPanel/index.tsx]: OK cool - good idea. Will dig into this.

File: src/components/TokenItem/index.tsx (hunk 2 above)
Comment (W3stside) [reply src/components/TokenItem/index.tsx]: no reason

**Comment by W3stside** [comment]
@Velenir can you quickly take a look at my buttonCTA changes? Cheers!

**Review by Velenir — APPROVED** [review]

**Pull request merged**